projects/core/styles/markup/tui-list.less

```diff
@@ -75,6 +75,20 @@
                 background-color: transparent;
             }
         }
+        
+        .tui-list_triangle & {
+            padding-left: @space * 7;
+
+            &:before {
+                content: '◤';
+                left: 0;
+                top: auto;
+                width: auto;
+                height: auto;
+                background-color: transparent;
+                color: #9299a2;
```

Comment:
```var(--tui-base-06)``` ?

Reply:
Thanks! It is pretty close to `#9299A2`, so I think, it would be better choise

Reply:
> it is pretty close to #9299A2

In our proprietary kit `--tui-base-06` is equal to `#9299A2` :D